functions/shuffle.go

```diff
@@ -0,0 +1,23 @@
+package functions
+
+import (
+	"math/rand"
+	"time"
+)
+
+// Max is the maximum value, or zero.
+func (ss SliceType) Shuffle() SliceType {
+	if len(ss) < 2 {
+		return ss
+	}
+
+	shuffled := make([]ElementType, len(ss))
+	copy(shuffled, ss)
+
+	rand.Seed(time.Now().UnixNano())
```

Comment:
I think shuffle should take a rand.Source or rand.Source64

Reply:
Thank you, that's a nice idea
I've changed it in this https://github.com/elliotchance/pie/pull/61/commits/157a787431917a8d653e30635eaebf799154444b commit